kubernetes/template/DeploymentTemplates/MainTemplate.json

```diff
@@ -147,6 +147,48 @@
         "description": "Network plugin which will deployed in Kubernetes cluster"
       },
       "type": "string"
+    },    
+    "windowsAdminUsername": {
```

Comment:
should we allow users to have different usernames for linux and windows? I think it may cause problems